### ISSUE TITLE
access a test dataone node for dev

### ISSUE THREAD

**Issue by jmlondon** [jmlondon/kotzeb0912#6]
initially, it might be preferable to get this use case working on a test/dev dataone server/node.

@mbjones is this possible? suggestions

**Comment by mbjones** [comment]
Yes, you can use the https://dev.nceas.ucsb.edu test server, which is not production but will allow you to submit test data and metadata.  Once everything is working as you expect, you can submit the real data to the KNB (https://knb.ecoinformatics.org) or the Arctic Data Center (https://arcticdata.io), or other DataONE repos as appropriate.
